website/docs/r/organization_token.html.markdown

```diff
@@ -28,6 +28,28 @@ The following arguments are supported:
 * `force_regenerate` - (Optional) If set to `true`, a new token will be
   generated even if a token already exists. This will invalidate the existing
   token!
+* `expired_at` - (Optional) The token's expiration date. The expiration date must be a date/time string in RFC3339 
+format (e.g., "2024-12-31T23:59:59Z"). If no expiration date is supplied, the expiration date will default to null and 
+never expire.
+
+## Example Usage
+
+When a token has an expiry:
+
+```hcl
+resource "tfe_organization_token" "test" {
+  organization = "my-org-name"
+}
+
```

Comment:
I think this is an extra resource defined in the example. If you are trying to show examples with and without an expired_at, that would be OK, but they should be separated by different code blocks maybe

```suggestion
```

Reply:
Whoops! That was an extra resource 🙈